### DISCUSSION TITLE
glad2 debug build recompiles every launch

### DISCUSSION THREAD

**Discussion by CuriousRubick** [Dav1dde/glad#450] [General]
Recently updated to glad2 and noticed that with the 'debug' option my program recompiles every source file that includes gl.h every time I run the program from vscode. Recompilation occurs even if I select "run without debugging." Happens in both linux and windows (gcc, clang and msvc). Non-debug works as expected.

Current options from https://gen.glad.sh/ : "c/c++, gl 4.1, core, all ext, alias, loader"

Any idea what's going on? 

*edit*

I link statically via cmake like this:

add_library(glad STATIC ${PATH}/src/gl.c)
target_link_libraries(${APP_NAME} glad)

Also linking glfw.

**Comment by Dav1dde** [comment]
Hey, I have no idea. There is nothing in glad which is conditional on debug/release either.

**Comment by CuriousRubick** [comment]
I believe I solved this problem. Looks like I was compiling the wrong mix of .h files due to not updating my gitignore when switching over to glad2.
